src/types/nitro.ts

```diff
@@ -61,10 +63,7 @@ export interface NitroConfig extends DeepPartial<NitroOptions> {
   extends?: string | string[] | NitroPreset
 }
 
-export interface NitroRouteOption {
-  swr?: boolean | number
-  redirect?: string
-}
+export type NitroRouteOption = Omit<CacheOptions<ResponseCacheEntry<unknown>>, 'getKey'>
```

Comment:
I think we can support more options as `{ cache }`. Route Options is not only for caching finally.

Reply:
Of course there will be more options in the future but swr is currently the only used option. Maybe instead of all the cache options we could just use `maxAge` and `swr` or make the `swr` option a string/number (eg. `'10 min'`/`600`, see this [Nuxt discussion](https://github.com/nuxt/framework/discussions/560))

Reply:
It was kinda experimental and indeed we can keep limited shortcuts top level but for the purpose of this PR to allow all cache related options, namespace seems more safe against naming conflict and semantical way of writing rules.